features/test_feature.py

```diff
@@ -270,6 +279,8 @@ def run_test():
         return True
     if test_config.current_test.startswith(('text_experimental_obj_', 'graphics_interop_')) and test_config.device_filter.count("cuda") == 0:
         return True
+    if test_config.current_test.startswith(('nvshmem')):
+        return True
```

Comment:
If line67 `'nvshmem'` is removed, line282~line283 is not needed.

Reply:
Done